modules/cnf-understanding-siteconfig-operator-flow.adoc

```diff
@@ -0,0 +1,23 @@
+// Module included in the following assemblies:
+// Epic CNF-9657 (TELCODOCS-1540) (4.16)
+// * edge_computing/cnf-understanding-siteconfig-operator.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="cnf-understanding-siteconfig-operator-flow_{context}"]
+= Understanding the {sco} flow
+
+The {sco} dynamically generates installation manifests based on user-defined cluster templates that are instantiated from the data that is encapsulated in the `ClusterInstance` CR.
```

Comment:
Perhaps make the sentence a bit less complex to aid readability?
"that are instantiated from the data in the `ClusterInstance` CR."